### PR TITLE
Add zookeeper-observ-lib

### PR DESCRIPTION
**Pull request by v-zhuravlev**
- Add zookeeper-observ-lib and uses it inside kafka-observ-lib/kafka-mixin
- Drop old zookeeper dashboard from kafka-mixin
- Add support for metricsSource: prometheus_old in jvm-observ-lib. This is metric style used in prometheus instrumentation prior to 1.0.0 version (in zookeeper metrics for example)
- Add support for collapsible rows, thanks to https://github.com/grafana/grafonnet/pull/214

Zookeeper dashboard in Kafka now looks like this:
image

With all JVM related rows imported from jvm-observ-lib but collapsed, to let users focus on zookeeper metrics by default: 
https://github.com/grafana/jsonnet-libs/pull/1220

### CODE REVIEW

**Review by gaantunes — APPROVED** [review]
LGTM. :rocket:

File: jvm-observ-lib/signals/hikari.libsonnet [hunk 1]
```diff
@@ -9,7 +9,8 @@ function(this)
     aggFunction: 'avg',
     discoveryMetric: {
       java_micrometer: 'hikaricp_connections',  // https://github.com/brettwooldridge/HikariCP/blob/dev/src/main/java/com/zaxxer/hikari/metrics/micrometer/MicrometerMetricsTracker.java
-      prometheus: '?',
+      // prometheus: '?',
+      // prometheus_old: '?',
```
Comment (Dasomeone) [inline jvm-observ-lib/signals/hikari.libsonnet:13]: Just a question here, if we're not using these discovery metrics, why have they been added to and then commented out?

File: kafka-observ-lib/config.libsonnet [hunk 2]
```diff
@@ -1,16 +1,17 @@
 {
   local this = self,
   filteringSelector: 'job!=""',
-  groupLabels: ['kafka_cluster'],
```
Comment (Dasomeone) [inline kafka-observ-lib/config.libsonnet:4]: I'm not 100% on this change. `cluster` is a reserved label for use with kubernetes clusters, which is why we changed this to `kafka_cluster` in the past to support multi-cluster kafka separately from kubernetes clusters.

File: kafka-observ-lib/config.libsonnet [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
   uid: 'kafka',
-  dashboardNamePrefix: 'Kafka',
+  dashboardNamePrefix: '',
   dashboardTags: ['kafka'],
   alertKafkaLagTooHighThreshold: '100',
   alertKafkaLagTooHighSeverity: 'critical',
   metricsSource: 'prometheus',  //or grafanacloud
   //Can be regex:
   topicsIgnoreSelector: '__consumer_offsets',
+  zookeeperEnabled: false,
```
Comment (Dasomeone) [inline kafka-observ-lib/config.libsonnet:14]: Do we not want to enable Zookeeper by default?

**Review by Dasomeone — COMMENTED** [review]
Couple comments, but nothing major :)

File: kafka-observ-lib/config.libsonnet (hunk 3 above)
Comment (v-zhuravlev) [reply kafka-observ-lib/config.libsonnet:14]: you are right, changing.

File: jvm-observ-lib/signals/hikari.libsonnet (hunk 1 above)
Comment (v-zhuravlev) [reply jvm-observ-lib/signals/hikari.libsonnet:13]: wanted to highlight that haven't found them. Removing for cleanliness.

**Pull request merged**